specyfikacje_i_cennik.txt

```diff
@@ -0,0 +1,37 @@
+
+SZCZEGÓŁOWE SPECYFIKACJE ORAZ CENNIK USŁUG
+
+Dysponujemy 24 biurkami: jedno, cztero i dziesięcioosobowymi, co oznacza łącznie 90 stanowisk pracy.
+W zależności od wybranej opcji oferujemy dodatkowy monitor.
+Stanowiska zostały tak zaprojektowane, aby zapewnić komfort i bezpieczeństwo pracy.
+Oferujemy sieć o prędkości do 1000 Mpbs, która zaspokaja potrzeby najbardziej wymagających klientów.
+Kuchnia wyposażona jest w czajnik, ekspres do kawy, mikrofalówkę, lodówkę oraz podstawowe naczynia.
+Produkty takie jak np. kawę, herbatę, cukier należy zorganizować we własnym zakresie. Nieograniczony dostęp do wody mineralnej zawiera się w cenie usługi.
+Poniżej znajduje się lista oferowanych biurek/stanowisk. Dostępność znajduje sie w zakładce DOSTĘPNOŚĆ BIUREK/STANOWISK.
+Szczegółowy cennik drukarki i ksero znajduje się w biurze.
+
+1. Biurko nr 1: Biurko pojedyncze bez monitora, trzy gniazdka, cena 50 PLN
```

Comment:
to powinno byc z  bazy zczytywane , jak ktos doda teraz nwoe biurko to przeciez nie pojawi sie on w tym pliku, tak samo ze zmiana nazyw, ceny itd

Reply:
No pewnie, znow zpomniałam , że to będzie sie zmieniać
:) Już dogadałam się z chłopakami, że zmodyfikujemy nasza główną bazę i z niej będę wyciągać potrzebne tu informacje